### ISSUE TITLE
KSU tools Add Tracks is ignoring rounded/filleted track segments

### ISSUE THREAD

**Issue by okelk** [easyw/kicadStepUpMod#171]
I want to import the tracks in 3d from a kicad 7 file that has rounded tracks.
rounded segments are ignored. straight segments get imported.

**Comment by easyw** [comment]
please post a minimal working example in pcb v7

**Comment by easyw** [comment]
rounded tracks are loaded, rounded drawings in copper layers are ignored... 
closing it ATM
feel free to reopen when you have a MWE

**Comment by okelk** [comment]
I attached a MWE kicad project as a zip.
also here some screenshotz that illustrate the problem:

![stepupbugfreecad](https://user-images.githubusercontent.com/8890607/236938954-92514176-c2f6-4b9c-9fd8-db06aa4a0001.JPG)
![stepupbugpcb](https://user-images.githubusercontent.com/8890607/236938961-66665b42-30fd-47bb-b0e9-d0b217f6a7ae.JPG)
[stepup-rounded.zip](https://github.com/easyw/kicadStepUpMod/files/11425279/stepup-rounded.zip)

**Comment by easyw** [comment]
no issue on my system... which is your kSU version?
From my FC Report panel printing:
`KiCadStepUpWB.Activated(v 10.18.8)`
`FC Version 021-0`
`kicad_parser_version 2.2.4`
`tracks version: 2.5.8`
`kicad version=20221018`

**Comment by okelk** [comment]
I found the issue.

The Addon Manager told me that I'm on **Version 10.18.8**  .
...so that's what I thought I'm using

But under
Edit --> Preferences --> KicadStepupGUI
kicad StepUp version **9.7.3.9**
was shown.

So I assume there was some conflict with some older version of the plugin?

I deleted the /Mod/kicadStepUpMod.bak folder and reinstalled the plugin via the Addon Manager.
Now it works.

Thank You!
